lib/main.js

```diff
@@ -18,16 +18,40 @@ function HTMLUglify(config) {
   this.whitelist = config.whitelist || [];
 }
 
+HTMLUglify.prototype.checkForCompoundPointer = function(lookups, value) {
+  var pointer;
+  var keys = Object.keys(lookups);
+
+  keys.forEach(function(key) {
+    var keyValue = key.split('=')[1];
+
+    if (value.indexOf(keyValue) !== -1) {
```

Comment:
Ok, I've added tests. The gist is this needs to actually match on the substring. This is checking for a scenario like:

```
<style>
*[class*=ever] { }
</style>
<body>
  <div class="everything"></div>
</body>
```

It ends up using the original generated lookup key as part of the html element's class name.

The lookup ends up looking like this.
```
{
  class: {
    ever: "xz"
  }
}
```

And the uglified html then ends up like this:

```
<style>
*[class*=xz] { }
</style>
<body>
  <div class="xzthing"></div>
</body>
```

Things are still uglified this way but the attribute selector matching still works.

I've added new tests to help highlight this and can add more. It's now named `#checkForAttributePointer`.